src/utils/BackendClient/NimbledroidApiHandler.js

```diff
@@ -0,0 +1,103 @@
+import fetchJson from '../fetchJson';
+
+const matchSiteName = profileName => profileName
+  .replace(/.*.*http[s]*:\/\/[www]*\.*(.*)[/].*/, (match, firstMatch) => firstMatch);
+
+const scenarioShouldBeConsidered = ({ scenario_name, status }) => {
+  if (
+    status !== 'fail' &&
+    scenario_name.startsWith('customFlow') &&
+    scenario_name.includes('http')
+  ) {
+    return true;
+  }
+  return false;
+};
```

Comment:
I don't think defining this as an external function helps with clarity. I would put it inline, maybe with a reference to any documentation on why this criteria we have available (or just say why in the comment, if we have no such documentation).